R/MzTabReader.R

```diff
@@ -125,6 +132,8 @@ getSummary = function()
 
   ## read all custom entries
   mtd_custom_df = .self$sections$MTD[grep("^custom", .self$sections$MTD$key), ]
+  
+  if (nrow(mtd_custom_df) == 0) return(NULL)
```

Comment:
if MS2 id-rate is not provided, we could guess it by looking at `PSM$spectra_ref`s largest index, and assume this to be the number of MS2 scans (per Raw file)